src/Build.OM.UnitTests/ObjectModelRemoting/RemoteProjectsProviderMock/CollectionsHelpers.cs

```diff
@@ -85,13 +85,13 @@ public static IList<MockProjectElementLinkRemoter> ExportCollection<T>(this Proj
         }
 
 
-        public static IDictionary<key, T> ImportDictionary<key, T, RMock>(this ProjectCollectionLinker importer, IDictionary<key, RMock> source)
+        public static IDictionary<Key, T> ImportDictionary<Key, T, RMock>(this ProjectCollectionLinker importer, IDictionary<Key, RMock> source)
```

Comment:
What motivated this change? Should it be `TKey` instead?

Reply:
The issue was "don't declare types like this as lowercase" so I just uppercased it. If TKey is what we should, thumbs up and I can make that change

Reply:
Yeah, I agree with this guideline: https://stackoverflow.com/a/1098058